UM/Mesh/MeshBuilder.py

```diff
@@ -34,6 +39,15 @@ def __init__(self):
         self._file_name = None
         # original center position
         self._center_position = None
+        
+        # File-changed notify timer
+        self._file_changed_timer = QTimer()
+        self._file_changed_timer.setInterval(1000)
+        self._file_changed_timer.setSingleShot(True)
+        self._file_changed_timer.timeout.connect(self._check_file_changed)
+        if self._file_name:
```

Comment:
Since self._file_name is set to None 9 lines above this, the timer never gets started. Right?

Reply:
Nope, it shouldn't. I guess we need for the case, that someone sets up the filename after initializing this class, something like a .setFilename() function, which starts the timer after setting the path to self._file_name.

Reply:
My point was that you can *only* set up the filename after the initialization.

Reply:
Didn't take a look at the def `__init__` line 😮
Will provide a function then.

Btw. how is the overall impression about the PR? Is the rest good?